react/features/analytics/handlers/amplitude/Amplitude.native.js

```diff
@@ -35,9 +35,12 @@ class Amplitude {
      * Sets an identifier for the current user.
      *
      * @param {string} userId - The new user id.
+     * @param {string} opt_userId - Currently not used.
```

Comment:
Why does it need to be there, if we are not using it?

Reply:
The web instance supports it and we are passing some of those. I guess to follow the interface...

Reply:
At least otherwise looks weird for me when I see somebody calling a function with more parameters than the defined ones...